src/sentry/lang/javascript/processor.py

```diff
@@ -57,7 +57,7 @@ class ZeroReturnError(Exception):
 )
 VERSION_RE = re.compile(r'^[a-f0-9]{32}|[a-f0-9]{40}$', re.I)
 NODE_MODULES_RE = re.compile(r'\bnode_modules/')
-SOURCE_MAPPING_URL_RE = re.compile(r'\/\/# sourceMappingURL=(.*)$')
+SOURCE_MAPPING_URL_RE = re.compile(r'\/\/# sourceMappingURL=([^<]*)$')
```

Comment:
How does this pattern change match `//# sourceMappingURL =https://xxxx/a.map`? The space between `URL =` would make this regex not match.

Reply:
The case above have a spell error. It should be:
```
<html><head><script>var a = 'hello';
//# sourceMappingURL=https://xxxx/a.map</script>
</head><body>...</body></html>
```

`\/\/# sourceMappingURL=(.*)$` would match to the end of line. But in this case, we should end it before `</script>`. 

I have digged into V8 source code, they do similar action like sentry but v8 only process js content. That means only content between `script` tags.

I also set up a  demo here:  https://my-react-app.o0x11.now.sh/. It would fail to download sourcemap when we use `r'\/\/# sourceMappingURL=(.*)$'` to extract sourceMappingURL.

Reply:
Thanks for the demo site. Your changes don't enable the source map URLs to be extracted from that response either though. Additional changes would need to be made as your output gets captured by the the conditionals on line 161-164.